packages/list-branches/action.yml

```diff
@@ -0,0 +1,32 @@
+name: 'List branches on repository'
+description: 'You can customize the message and the channel id.'
+inputs:
+  branche_regex:
+    description: 'The slack channel id'
+    required: true
+    default: '^master$|^dev$|release-.*'
+outputs:
+  branches:
+    description: "Json object who contains each branch"
+    value: ${{ steps.extract_branches.outputs.branches }}
+runs:
+  using: "composite"
+  steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Install jq and jo
+      shell: bash
+      run: |
+        sudo apt-get update
+        sudo apt-get install jq jo
+    - name: List branches
+      shell: bash
+      run: |
+        git remote set-branches origin '*'
+        git fetch
+        readarray branches <<< $(git branch --remote --format='%(refname:lstrip=3)' | tr '\n' ' ' | awk '{$1=$1};1')
+        json=$(jo -a $branches)
+        echo "##[set-output name=branches;]$(echo $json | jq -c 'map(select(test("${{inputs.branche_regex}}")))')"
```

Comment:
Regarding the suggestion above, try to transform the `selected_branches` into a regexp here